### ISSUE TITLE
www.haltadefinizione.com

### ISSUE THREAD

**Issue by juri1174** [lovasoa/dezoomify#833]
Hello everyone,

 I am having issues when trying to download https://www.haltadefinizione.com/assets/Remote/_resampled/ScaleHeightWyI2MDAiXQ/2a69ccfa-9c32-4e1b-885a-08bdee4ba0d7.jpeg

Dezoomify reports:

```
Uncaught Error: Invalid XML:
https://www.haltadefinizione.com/assets/Remote/_resampled/ScaleHeightWyI2MDAiXQ/2a69ccfa-9c32-4e1b-885a-08bdee4ba0d7.jpeg

(https://dezoomify.ophir.dev/zoommanager.js:221)
```
I don't understand this message. Can someone please help me ?

**Comment by Muddysoil** [comment]
Hi!

This is the xml file you need in order to download the image:
https://dam.haltadefinizione.com/images/dzi-backend/db/2a69ccfa-9c32-4e1b-885a-08bdee4ba0d7/file.dzi

Copy this file into the dezoomify-rs and choose which resolution you want to download.

Happy New Year!

**Comment by juri1174** [comment]
niente non riesco a scaricare i files.
da altri siti riesco a trasportare il file ma da qui no, forse dovrei usare dezoomify-rs?

**Comment by applecuckoo** [comment]
yes, use dezoomify-rs, it works better with large images like this

**Comment by lovasoa** [comment]
Closing as a duplicate site report for haltadefinizione.com. The oldest open tracking issue I found for this website is #208; please keep follow-up investigation for this site there.
